### PR TITLE
Add "flip-back" action

### PR DESCRIPTION
**Pull request by ptomato**
Some commits from the framework hacking branch that can already be reviewed.

https://phabricator.endlessm.com/T23625

### CODE REVIEW

File: com.endlessm.HackToolbox.json.in [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "app-id": "com.endlessm.HackToolbox",
-  "runtime": "org.gnome.Platform",
-  "runtime-version": "3.28",
+  "runtime": "com.endlessm.apps.Sdk",
+  "runtime-version": "master",
```
Comment (cosimoc) [inline com.endlessm.HackToolbox.json.in]: Did you intentionally set this to `master`? I was expecting we'd use `5`

File: src/app.js [hunk 2]
```diff
@@ -76,4 +83,24 @@ var HackToolboxApplication = GObject.registerClass(class extends Gtk.Application
...
     }
+
+    _onFlipBack(action, parameterVariant) {
+        this.hold();
+        const unpacked = parameterVariant.deep_unpack();
+        const [busName, objectPath] = unpacked;
+        log(`Call flip-back with ${JSON.stringify(unpacked)}`);
+
+        const win = this._windows[busName][objectPath];
+        if (win) {
```
Comment (cosimoc) [inline src/app.js:94]: In the case this check fails, we won't call `release()`.

**Review by cosimoc — CHANGES_REQUESTED** [review]

File: com.endlessm.HackToolbox.json.in (hunk 1 above)
Comment (ptomato) [reply com.endlessm.HackToolbox.json.in]: Yes, makes sense.

File: src/app.js (hunk 2 above)
Comment (ptomato) [reply src/app.js:94]: Fixed below.

**Pull request merged**